package.json

```diff
@@ -23,6 +23,7 @@
     "lint-staged": "^9.5.0",
     "lodash.throttle": "^4.1.1",
     "prettier": "^1.19.1",
+    "query-string": "^6.11.1",
```

Comment:
This should've updated the `package-lock.json`. Can you please push up the changes?